### ISSUE TITLE
使用官方提供的训练模型和对应的配置文件，

### ISSUE THREAD

**Issue by zikaixie** [PaddlePaddle/PaddleOCR#2868]
模型：ch_ppocr_mobile_v2.0_det

配置文件：https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.1/configs/det/ch_ppocr_v2.0/ch_det_mv3_db_v2.0.yml

测试命令：python tools/infer_det.py -c configs/det/ch_det_mv3_db_v2.0.yml -o Global.infer_img="./doc/imgs_en/img_12.j
pg" Global.pretrained_model="./output/det_mv3_db_v2.0_train/best_accuracy"

测试结果：
![img_10](https://user-images.githubusercontent.com/62594309/119111115-58384000-ba55-11eb-9782-cb90257df3bf.jpg)
下面的几个小方块
![img_12](https://user-images.githubusercontent.com/62594309/119111154-5ff7e480-ba55-11eb-89c8-0de5b61df08e.jpg)
右边边缘的几个小方块

```yml
Global:
  use_gpu: true
  epoch_num: 1200
  log_smooth_window: 20
  print_batch_step: 2
  save_model_dir: ./output/ch_db_mv3/
  save_epoch_step: 1200
  # evaluation is run every 5000 iterations after the 4000th iteration
  eval_batch_step: [3000, 2000]
  cal_metric_during_train: False
  pretrained_model: ./pretrain_models/MobileNetV3_large_x0_5_pretrained
  checkpoints:
  save_inference_dir:
  use_visualdl: False
  infer_img: doc/imgs_en/img_10.jpg
  save_res_path: ./output/det_db/predicts_db.txt

Architecture:
  model_type: det
  algorithm: DB
  Transform:
  Backbone:
    name: MobileNetV3
    scale: 0.5
    model_name: large
    disable_se: True
  Neck:
    name: DBFPN
    out_channels: 96
  Head:
    name: DBHead
    k: 50

Loss:
  name: DBLoss
  balance_loss: true
  main_loss_type: DiceLoss
  alpha: 5
  beta: 10
  ohem_ratio: 3

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.001
    warmup_epoch: 2
  regularizer:
    name: 'L2'
    factor: 0

PostProcess:
  name: DBPostProcess
  thresh: 0.3
  box_thresh: 0.6
  max_candidates: 1000
  unclip_ratio: 1.5

Metric:
  name: DetMetric
  main_indicator: hmean

Train:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/icdar2015/text_localization/
    label_file_list:
      - ./train_data/icdar2015/text_localization/train_icdar2015_label.txt
    ratio_list: [1.0]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - DetLabelEncode: # Class handling label
      - IaaAugment:
          augmenter_args:
            - { 'type': Fliplr, 'args': { 'p': 0.5 } }
            - { 'type': Affine, 'args': { 'rotate': [-10, 10] } }
            - { 'type': Resize, 'args': { 'size': [0.5, 3] } }
      - EastRandomCropData:
          size: [960, 960]
          max_tries: 50
          keep_ratio: true
      - MakeBorderMap:
          shrink_ratio: 0.4
          thresh_min: 0.3
          thresh_max: 0.7
      - MakeShrinkMap:
          shrink_ratio: 0.4
          min_text_size: 8
      - NormalizeImage:
          scale: 1./255.
          mean: [0.485, 0.456, 0.406]
          std: [0.229, 0.224, 0.225]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: ['image', 'threshold_map', 'threshold_mask', 'shrink_map', 'shrink_mask'] # the order of the dataloader list
  loader:
    shuffle: True
    drop_last: False
    batch_size_per_card: 8
    num_workers: 4

Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/icdar2015/text_localization/
    label_file_list:
      - ./train_data/icdar2015/text_localization/test_icdar2015_label.txt
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - DetLabelEncode: # Class handling label
      - DetResizeForTest:
#           image_shape: [736, 1280]
      - NormalizeImage:
          scale: 1./255.
          mean: [0.485, 0.456, 0.406]
          std: [0.229, 0.224, 0.225]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: ['image', 'shape', 'polys', 'ignore_tags']
  loader:
    shuffle: False
    drop_last: False
    batch_size_per_card: 1 # must be 1
    num_workers: 2
```

**Comment by WenmuZhou** [comment]
贴一下日志吧

**Comment by zikaixie** [comment]
> 贴一下日志吧

(base) C:\Users\shibi\PycharmProjects\PaddleOCR>python tools/infer_det.py -c configs/det/ch_det_mv3_db_v2.0.yml -o Global.infer_img="./doc/imgs_en/img_12.j
pg" Global.pretrained_model="./output/det_mv3_db_v2.0_train/best_accuracy"
C:\ProgramData\Anaconda3\lib\site-packages\win32\lib\pywintypes.py:2: DeprecationWarning: the imp module is deprecated in favour of importlib; see the modu
le's documentation for alternative uses
  import imp, sys, os
[2021/05/21 16:52:28] root INFO: Architecture :
[2021/05/21 16:52:28] root INFO:     Backbone :
[2021/05/21 16:52:28] root INFO:         disable_se : True
[2021/05/21 16:52:28] root INFO:         model_name : large
[2021/05/21 16:52:28] root INFO:         name : MobileNetV3
[2021/05/21 16:52:28] root INFO:         scale : 0.5
[2021/05/21 16:52:28] root INFO:     Head :
[2021/05/21 16:52:28] root INFO:         k : 50
[2021/05/21 16:52:28] root INFO:         name : DBHead
[2021/05/21 16:52:28] root INFO:     Neck :
[2021/05/21 16:52:28] root INFO:         name : DBFPN
[2021/05/21 16:52:28] root INFO:         out_channels : 96
[2021/05/21 16:52:28] root INFO:     Transform : None
[2021/05/21 16:52:28] root INFO:     algorithm : DB
[2021/05/21 16:52:28] root INFO:     model_type : det
[2021/05/21 16:52:28] root INFO: Eval :
[2021/05/21 16:52:28] root INFO:     dataset :
[2021/05/21 16:52:28] root INFO:         data_dir : ./train_data/icdar2015/text_localization/
[2021/05/21 16:52:28] root INFO:         label_file_list : ['./train_data/icdar2015/text_localization/test_icdar2015_label.txt']
[2021/05/21 16:52:28] root INFO:         name : SimpleDataSet
[2021/05/21 16:52:28] root INFO:         transforms :
[2021/05/21 16:52:28] root INFO:             DecodeImage :
[2021/05/21 16:52:28] root INFO:                 channel_first : False
[2021/05/21 16:52:28] root INFO:                 img_mode : BGR
[2021/05/21 16:52:28] root INFO:             DetLabelEncode : None
[2021/05/21 16:52:28] root INFO:             DetResizeForTest : None
[2021/05/21 16:52:28] root INFO:             NormalizeImage :
[2021/05/21 16:52:28] root INFO:                 mean : [0.485, 0.456, 0.406]
[2021/05/21 16:52:28] root INFO:                 order : hwc
[2021/05/21 16:52:28] root INFO:                 scale : 1./255.
[2021/05/21 16:52:28] root INFO:                 std : [0.229, 0.224, 0.225]
[2021/05/21 16:52:28] root INFO:             ToCHWImage : None
[2021/05/21 16:52:28] root INFO:             KeepKeys :
[2021/05/21 16:52:28] root INFO:                 keep_keys : ['image', 'shape', 'polys', 'ignore_tags']
[2021/05/21 16:52:28] root INFO:     loader :
[2021/05/21 16:52:28] root INFO:         batch_size_per_card : 1
[2021/05/21 16:52:28] root INFO:         drop_last : False
[2021/05/21 16:52:28] root INFO:         num_workers : 2
[2021/05/21 16:52:28] root INFO:         shuffle : False
[2021/05/21 16:52:28] root INFO: Global :
[2021/05/21 16:52:28] root INFO:     cal_metric_during_train : False
[2021/05/21 16:52:28] root INFO:     checkpoints : None
[2021/05/21 16:52:28] root INFO:     debug : False
[2021/05/21 16:52:28] root INFO:     distributed : False
[2021/05/21 16:52:28] root INFO:     epoch_num : 1200
[2021/05/21 16:52:28] root INFO:     eval_batch_step : [3000, 2000]
[2021/05/21 16:52:28] root INFO:     infer_img : ./doc/imgs_en/img_12.jpg
[2021/05/21 16:52:28] root INFO:     log_smooth_window : 20
[2021/05/21 16:52:28] root INFO:     pretrained_model : ./output/det_mv3_db_v2.0_train/best_accuracy
[2021/05/21 16:52:28] root INFO:     print_batch_step : 2
[2021/05/21 16:52:28] root INFO:     save_epoch_step : 1200
[2021/05/21 16:52:28] root INFO:     save_inference_dir : None
[2021/05/21 16:52:28] root INFO:     save_model_dir : ./output/ch_db_mv3/
[2021/05/21 16:52:28] root INFO:     save_res_path : ./output/det_db/predicts_db.txt
[2021/05/21 16:52:28] root INFO:     use_gpu : True
[2021/05/21 16:52:28] root INFO:     use_visualdl : False
[2021/05/21 16:52:28] root INFO: Loss :
[2021/05/21 16:52:28] root INFO:     alpha : 5
[2021/05/21 16:52:28] root INFO:     balance_loss : True
[2021/05/21 16:52:28] root INFO:     beta : 10
[2021/05/21 16:52:28] root INFO:     main_loss_type : DiceLoss
[2021/05/21 16:52:28] root INFO:     name : DBLoss
[2021/05/21 16:52:28] root INFO:     ohem_ratio : 3
[2021/05/21 16:52:28] root INFO: Metric :
[2021/05/21 16:52:28] root INFO:     main_indicator : hmean
[2021/05/21 16:52:28] root INFO:     name : DetMetric
[2021/05/21 16:52:28] root INFO: Optimizer :
[2021/05/21 16:52:28] root INFO:     beta1 : 0.9
[2021/05/21 16:52:28] root INFO:     beta2 : 0.999
[2021/05/21 16:52:28] root INFO:     lr :
[2021/05/21 16:52:28] root INFO:         learning_rate : 0.001
[2021/05/21 16:52:28] root INFO:         name : Cosine
[2021/05/21 16:52:28] root INFO:         warmup_epoch : 2
[2021/05/21 16:52:28] root INFO:     name : Adam
[2021/05/21 16:52:28] root INFO:     regularizer :
[2021/05/21 16:52:28] root INFO:         factor : 0
[2021/05/21 16:52:28] root INFO:         name : L2
[2021/05/21 16:52:28] root INFO: PostProcess :
[2021/05/21 16:52:28] root INFO:     box_thresh : 0.6
[2021/05/21 16:52:28] root INFO:     max_candidates : 1000
[2021/05/21 16:52:28] root INFO:     name : DBPostProcess
[2021/05/21 16:52:28] root INFO:     thresh : 0.3
[2021/05/21 16:52:28] root INFO:     unclip_ratio : 1.5
[2021/05/21 16:52:28] root INFO: Train :
[2021/05/21 16:52:28] root INFO:     dataset :
[2021/05/21 16:52:28] root INFO:         data_dir : ./train_data/icdar2015/text_localization/
[2021/05/21 16:52:28] root INFO:         label_file_list : ['./train_data/icdar2015/text_localization/train_icdar2015_label.txt']
[2021/05/21 16:52:28] root INFO:         name : SimpleDataSet
[2021/05/21 16:52:28] root INFO:         ratio_list : [1.0]
[2021/05/21 16:52:28] root INFO:         transforms :
[2021/05/21 16:52:28] root INFO:             DecodeImage :
[2021/05/21 16:52:28] root INFO:                 channel_first : False
[2021/05/21 16:52:28] root INFO:                 img_mode : BGR
[2021/05/21 16:52:28] root INFO:             DetLabelEncode : None
[2021/05/21 16:52:28] root INFO:             IaaAugment :
[2021/05/21 16:52:28] root INFO:                 augmenter_args :
[2021/05/21 16:52:28] root INFO:                     args :
[2021/05/21 16:52:28] root INFO:                         p : 0.5
[2021/05/21 16:52:28] root INFO:                     type : Fliplr
[2021/05/21 16:52:28] root INFO:                     args :
[2021/05/21 16:52:28] root INFO:                         rotate : [-10, 10]
[2021/05/21 16:52:28] root INFO:                     type : Affine
[2021/05/21 16:52:28] root INFO:                     args :
[2021/05/21 16:52:28] root INFO:                         size : [0.5, 3]
[2021/05/21 16:52:28] root INFO:                     type : Resize
[2021/05/21 16:52:28] root INFO:             EastRandomCropData :
[2021/05/21 16:52:28] root INFO:                 keep_ratio : True
[2021/05/21 16:52:28] root INFO:                 max_tries : 50
[2021/05/21 16:52:28] root INFO:                 size : [960, 960]
[2021/05/21 16:52:28] root INFO:             MakeBorderMap :
[2021/05/21 16:52:28] root INFO:                 shrink_ratio : 0.4
[2021/05/21 16:52:28] root INFO:                 thresh_max : 0.7
[2021/05/21 16:52:28] root INFO:                 thresh_min : 0.3
[2021/05/21 16:52:28] root INFO:             MakeShrinkMap :
[2021/05/21 16:52:28] root INFO:                 min_text_size : 8
[2021/05/21 16:52:28] root INFO:                 shrink_ratio : 0.4
[2021/05/21 16:52:28] root INFO:             NormalizeImage :
[2021/05/21 16:52:28] root INFO:                 mean : [0.485, 0.456, 0.406]
[2021/05/21 16:52:28] root INFO:                 order : hwc
[2021/05/21 16:52:28] root INFO:                 scale : 1./255.
[2021/05/21 16:52:28] root INFO:                 std : [0.229, 0.224, 0.225]
[2021/05/21 16:52:28] root INFO:             ToCHWImage : None
[2021/05/21 16:52:28] root INFO:             KeepKeys :
[2021/05/21 16:52:28] root INFO:                 keep_keys : ['image', 'threshold_map', 'threshold_mask', 'shrink_map', 'shrink_mask']
[2021/05/21 16:52:28] root INFO:     loader :
[2021/05/21 16:52:28] root INFO:         batch_size_per_card : 8
[2021/05/21 16:52:28] root INFO:         drop_last : False
[2021/05/21 16:52:28] root INFO:         num_workers : 4
[2021/05/21 16:52:28] root INFO:         shuffle : True
[2021/05/21 16:52:28] root INFO: train with paddle 2.0.1 and device CUDAPlace(0)
W0521 16:52:28.341244 12656 device_context.cc:362] Please NOTE: device: 0, GPU Compute Capability: 6.1, Driver API Version: 11.3, Runtime API Version: 10.2

W0521 16:52:28.360229 12656 device_context.cc:372] device: 0, cuDNN Version: 7.6.
[2021/05/21 16:52:31] root INFO: load pretrained model from ['./output/det_mv3_db_v2.0_train/best_accuracy']
[2021/05/21 16:52:31] root INFO: infer_img: ./doc/imgs_en/img_12.jpg
[2021/05/21 16:52:31] root INFO: The detected Image saved in ./output/det_db/det_results/img_12.jpg
[2021/05/21 16:52:31] root INFO: success!

**Comment by WenmuZhou** [comment]
没有复现你的问题
![image](https://user-images.githubusercontent.com/12406017/119121666-44dea200-ba60-11eb-8600-de5b3029759c.png)
![image](https://user-images.githubusercontent.com/12406017/119121759-6049ad00-ba60-11eb-9c72-8b53192f1235.png)
 命令是 python3 tools/infer_det.py -c configs/det/det_mv3_db.yml -o Global.infer_img="./doc/imgs_en/img_10.jpg" Global.pretrained_model=det_mv3_db_v2.0_train/best_accuracy
模型是 https://paddleocr.bj.bcebos.com/dygraph_v2.0/en/det_mv3_db_v2.0_train.tar
